### PR TITLE
Update FAQ with Rescan LND

### LINKED ISSUE #1106: Add documentation on howto rescan the LND onchain wallet
Leaving this here as a reminder that you wanted to add this to documentation.

Here is the proposed opt-lnd-reset-onchain.yml which @NicolasDorier wanted in the docu (not sure where it would fit):

```
# only enable this fragment temporarly to reset the onchain wallet transactions of lnd and to trigger a rescan
# after a successfull rescan, you should disable this fragment again, or it will rescan the onchain wallet at
# every restart!
#
# WARNING: The rescan can only pickup the transactions that are within your archived blocks (beware of PRUNED nodes!)
version: "3"

services:
  lnd_bitcoin:
    environment:
      LND_EXTRA_ARGS: |
        reset-wallet-transactions=1
  lnd_litecoin:
    environment:
      LND_EXTRA_ARGS: |
        reset-wallet-transactions=1
  lnd_bitcoingold:
    environment:
      LND_EXTRA_ARGS: |
        reset-wallet-transactions=1
```

Here is the closed pull request: https://github.com/btcpayserver/btcpayserver-docker/pull/548

### PR DESCRIPTION
**Pull request by Bangalisch**
Yo,

As posted by @petzsch there was a request to add this information on docs.
I felt FAQ would be suiting for it.

closes  #1106

### CODE REVIEW

File: docs/FAQ/LightningNetwork.md [hunk 1]
```diff
@@ -293,6 +293,33 @@ sudo su -
...
 
+### How to rescan the LND on-chain wallet?
+
+:::warning
+only enable this fragment temporarily to reset the on-chain wallet transactions of lnd and to trigger a rescan
+after a successful rescan, you should disable this fragment again, or it will rescan the on-chain wallet at
+every restart!
+WARNING: The rescan can only pick up the transactions within your archived blocks (beware of PRUNED nodes!)
+:::
+
```
Comment (petzsch) [inline docs/FAQ/LightningNetwork.md:304]:
```suggestion
To customize LND settings which are not available as environment variables, you can [create a custom fragment](../Docker/README.md#how-can-i-customize-the-generated-docker-compose-file) in `docker-compose-generator/docker-fragments/opt-lnd-wallet-rescan.custom.yml` like this:
```

File: docs/FAQ/LightningNetwork.md [hunk 2]
```diff
@@ -293,6 +293,33 @@ sudo su -
...
+  lnd_litecoin:
+    environment:
+      LND_EXTRA_ARGS: |
+        reset-wallet-transactions=1
+  lnd_bitcoingold:
+    environment:
+      LND_EXTRA_ARGS: |
+        reset-wallet-transactions=1
+```
+
```
Comment (petzsch) [inline docs/FAQ/LightningNetwork.md:322]:
```suggestion
For more information on this LND feature, see [their documentation](https://github.com/lightningnetwork/lnd/blob/master/docs/recovery.md#forced-in-place-rescan).
```

**Review by petzsch — CHANGES_REQUESTED** [review]
Had 2 additions to make the use of the code block more clear. :-)

**Comment by Bangalisch** [comment]
Should be good now :) @pavlenex

**Pull request merged**